webcompat/views.py

```diff
@@ -274,7 +275,16 @@ def contributors():
 @app.route('/tools/cssfixme')
 def cssfixme():
     '''Route for CSS Fix me tool'''
-    return render_template('cssfixme.html')
+
+    url = request.args.get('url')
+    escaped_csscode = ''
+    if url:
+        req = requests.get(url)
```

Comment:
Probably is not that much if url than probably a `try:` and if the request fails act accordingly.